src/AddIns/Uno.UI.MSAL/Uno.UI.MSAL.Wasm.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
This won't likely pass the build. Why do you need to remove it ?

Reply:
It is duplicate, as the same already comes from `<Import Project="../../netcore-build.props"/>`

Reply:
Similar for the other changes with `net5.0-windows` removal